.github/ISSUE_TEMPLATE.md

```diff
@@ -1,8 +1,8 @@
-<!-- This issue tracker is only for technical issues related to Bitcoin Core.
+<!-- This issue tracker is only for technical issues related to Dimecoin Core.
 
-General bitcoin questions and/or support requests are best directed to the Bitcoin StackExchange at https://bitcoin.stackexchange.com.
+General Dimecoin questions and/or support requests are best directed to the Dimecoin [Telegram Support Channel](https://t.me/dimeofficialsupport) or contact via [email](mailto:support@dimecoinnetwork.com).
 
-For reporting security issues, please read instructions at https://bitcoincore.org/en/contact/.
+For reporting security issues, please contact the Dimecoin developers via https://dimecoinnetwork.com or email directly at developer@dimecoinnetwork.com.
```

Comment:
You might not want to include an email address here.  Web scrapers will probably pick this up and spam.

Reply:
amended to avoid spam